composer.json

```diff
@@ -34,7 +34,7 @@
         "propel/propel1": "~1.7",
         "symfony/yaml": "2.*",
         "symfony/translation": "~2.0",
-        "symfony/validator": "~2.0",
+        "symfony/validator": "~2.0, <2.2",
```

Comment:
I corrected it in https://github.com/rpiotaix/serializer/tree/dependencies
Should i add it to this pull request or create a separate one for it?